projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -50,6 +50,7 @@ public function __construct( $connection_manager = null ) {
 			99
 		);
 		add_action( 'load-' . $page_suffix, array( $this, 'admin_init' ) );
+		add_action( 'jetpack_heartbeat', array( 'this', 'refresh_plans_data' ) );
```

Comment:
Nice!

Reply:
I actually removed this because jetpack_heartbeat runs only once a day

Reply:
I am instead refreshing the plan data just before fetching the current plan info in the `should_show_social_plan_nudge` method